scipy/signal/filter_design.py

```diff
@@ -4843,6 +4845,152 @@ def _design_notch_peak_filter(w0, Q, ftype, fs=2.0):
     return b, a
 
 
+def _hz_to_erb(hz):
+    """
+    Utility for converting from frequency (Hz) to the
+    Equivalent Rectangular Bandwith (ERB) scale
+    """
+    EarQ = 9.26449
+    minBW = 24.7
+    order = 1
+    return ((hz / EarQ) ** order + minBW ** order) ** (1 / order)
+
+
+def iirgammatone(freq, fs=16000, output='ba'):
```

Comment:
- I'm not sure it's worth having the `output` here. It's computed as `tf` form, it's easy enough (one-liner/wrapped) for people to convert at the end.
- There should either be no default `fs`, or it should be `1.` or `2 * np.pi` as in other places in the code. My vote here would probably be for no default